guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc

```diff
@@ -8,18 +8,18 @@ include::proc_creating-a-host-in-satellite.adoc[leveloffset=+1]
 
 ifeval::["{build}" == "satellite"]
 [id="editing-the-system-purpose-of-a-host"]
-== Editing the System Purpose of a Host
+== Editing the System Purpose of Hosts
 
-You can edit the system purpose attributes for a Red{nbsp}Hat Enterprise{nbsp}Linux 8 host. System purpose attributes define which subscriptions to attach automatically to this host. For more information about system purpose, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/graphical-installation_graphical-installation#configuring-system-purpose-standard_configuring-system-settings[Configuring system purpose] in the _Performing a standard RHEL installation_ guide.
+You can edit the system purpose attributes of Red{nbsp}Hat Enterprise{nbsp}Linux 8 hosts. System purpose attributes define which subscriptions to attach automatically to hosts. For more information about system purpose, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/graphical-installation_graphical-installation#configuring-system-purpose-standard_configuring-system-settings[Configuring system purpose] in the _Performing a standard RHEL installation_ guide.
 
 .Procedure
 
-. In the {Project} web UI, navigate to *Hosts* > *Content Hosts* and click the name of the Red{nbsp}Hat Enterprise Linux{nbsp}8 host that you want to edit.
-. In the *System Purpose* area, click the *Edit* or *Remove* icon for the system purpose attributes that you want to edit, add, or remove.
-. Click *Save*.
-. Click the *Subscriptions* tab and select *Subscriptions*.
-. Click *Run Auto-Attach* to attach subscriptions to your host automatically.
-. Refresh the page to verify that the subscriptions list contains the correct subscriptions.
+. In the {Project} web UI, navigate to *Hosts* > *Content Hosts* and select Red{nbsp}Hat Enterprise Linux{nbsp}8 hosts that you want to edit.
+. Click the *Select Action* button and from the drop-down menu, click *Manage System Purpose*.
+. Select the system purpose attributes that you want to assign to the selected hosts and click *Assign*.
+. Navigate to *Hosts* > *Content Hosts* and select the same Red{nbsp}Hat Enterprise Linux{nbsp}8 hosts to automatically attach subscriptions based on the system purpose.
+. Click the *Select Action* button and from the drop-down menu, click *Manage Subscriptions*.
+. Click *Auto-Attach* to attach subscriptions to all selected hosts automatically.
 
```

Comment:
maybe worth mentioning how to verify the bulk action succeeded from the actual hosts (in fact there was a bug precisely about this https://bugzilla.redhat.com/show_bug.cgi?id=1905979), commands to verify:

subscription-manager service-level
subscription-manager usage
subscription-manager role

Reply:
It is problematic to log in to all hosts, and ideally the procedure should just work, so I will kepp this verification to users.